rules/promotions/credential_access_endgame_cred_dumping_prevented.toml

```diff
@@ -19,6 +19,10 @@ language = "kuery"
 license = "Elastic License v2"
 max_signals = 10000
 name = "Credential Dumping - Prevented - Elastic Endgame"
+note = """## Setup
+
+The `max_signals` field is set to a value greater than the default value (1000) set by `system_limit`. This is to ensure that all alerts are captured.
```

Comment:
We usually add setup guides in the setup field.

Reply:
Good catch. I mixed up `config` vs `note` vs `setup`.